prow/plugins/updateconfig/README.md

```diff
@@ -13,7 +13,11 @@ plugins:
 config_updater:
   maps:
     # Update the whatever configmap whenever thing changes
-    path/to/some/other/thing: whatever
+    path/to/some/other/thing:
+      name: whatever
+    path/to/some/other/thing2:
+      name: whatever
+      namespace: nonDefaultNamespace
     # Update the config configmap whenever config.yaml changes
     prow/config.yaml: config
```

Comment:
All of these need to be updated to be valid config

Reply:
done